operations/helm/charts/mimir-distributed/templates/memcached/_memcached-statefulset.tpl

```diff
@@ -83,6 +87,11 @@ spec:
             - containerPort: {{ .port }}
               name: client
           args:
+            {{- if .args }}
```

Comment:
I feel like this puts an extra burden on the user to keep track of all the arguments.
Some alternatives:
* An `extraExtendedOptions` value to concatenate to `--extend` ?
* Make TLS config a top level value and set up everything automatically.

Reply:
I'll stick with the easier option of having extraExtendedOptions

Reply:
Verify locally first, but you can give Memcached multiple `--extended` flags. That's what we do for Memcached mTLS in jsonnet: https://github.com/grafana/mimir/blob/1171523e067a9a960240d064a6bc40066c4a0933/operations/mimir/memcached.libsonnet#L23-L46